jetty-core/jetty-server/src/test/java/org/eclipse/jetty/server/handler/ResourceHandlerTest.java

```diff
@@ -1941,6 +1957,9 @@ public void testEtagIfNoneMatchModifiedFile() throws Exception
         }
         while (Files.getLastModifiedTime(testFile).equals(before));
 
+        // The cached Resource will only go to fileSystem for expiryTime once per second.
```

Comment:
`expiryTime` is now called differently?
Also, I would not assume default values. Explicitly configure the period in `ValidatingCachingHttpContentFactory` with a large-ish value (e.g. `period = 2000`), and here wait for `1.5 * period`, so we don't make this test flaky for 100 ms.

Reply:
Its already configured at 0, just forgot to remove these sleeps.